configs/imagenet/efficientnet/efficientnet_b0.py

```diff
@@ -0,0 +1,67 @@
+# model settings
+model = dict(
```

Comment:
Because the input image size and augmentation strategies for each architecture of efficientnet are not the same. So there may be too many backbone and dataset configs in _base_. So, to make it easier for users to test or use our pretrained models, I think it's better to keep the existing format?

Reply:
Sounds reasonable to me. One thing is that PR https://github.com/open-mmlab/mmclassification/pull/16 will remove `gt_labels` in test_pipeline, and it may influence these configs.

Reply:
I can modify these configs after PR16 is merged.